client/components/settings/edit-access-modal.js

```diff
@@ -21,7 +27,9 @@ export default class EditAccessModal extends React.Component {
 
         let customFields;
         if (this.props.customFields) {
-            customFields = this.props.customFields.map(this.extractCustomFieldValue);
+            customFields = this.props.customFields.map(
+                (field, index) => this.extractCustomFieldValue(field, index)
```

Comment:
Haaa I get it. I'd prefer this code to not getting touched, and instead in the ctor that you bind the extractCustomFieldValue to `this` (as for `this.handleSubmit`). Do you mind changing this, please?